src/api/mod.rs

```diff
@@ -49,7 +49,36 @@ fn config_json_res(res: &mut Response) {
     res.set(MediaType::Json);
     res.set(StatusCode::Ok);
 }
+/*
+fn handle_db<'a>(req: &mut Request, mut res: Response<'a>) -> MiddlewareResult<'a> {
```

Comment:
can you explain what this is? In general, try to remove commented code (or at least give a reason why it should stay